src/netius/test/servers/socks.py

```
@@ -0,0 +1,204 @@
+            connection.parser.port = 8080
+
+            with mock.patch.object(
+                connection.parser, "get_address", return_value=b"\x7f\x00\x00\x01"
+            ):
+                with mock.patch.object(connection, "send") as send:
+                    connection.send_response_extra()
+
+            # the reply of the fifth version carries the address that was
+            # asked for and the port of it, after the status of the request
+            data = send.call_args[0][0]
+            version, status, _reserved, type = struct.unpack("!BBBB", data[:4])
```

Ruff (0.16.3)

[error] Variable `type` is shadowing a Python builtin

(A001)